### ISSUE TITLE
All error messages receive no focus, hence cannot be read by assistive technology (JAWS).

### ISSUE THREAD

**Issue by ltsheu** [18F/dol-whd-14c#536]
**Pages affected:** All

**Bug found by OCIO tester:** 
All error messages receive no focus, hence cannot be read by assistive technology (JAWS).

> Note: Guidance from OCIO tester is to make each error message receive tabbable focus.

**Solution:**
- [ ] Add the attribute `tabindex="0"` to each error message.

**Comment by ltsheu** [comment]
@davoodharun and I recommend that the work on this bug should wait until after the Login Workflow updates have been merged into the Master branch. After the Login Workflow is merged, the error message `tabindex` updates can be done on all pages, including the new pages in the Login Workflow. 

I am moving this bug #536 to blocked until after the Login Workflow updates are merged.

**Comment by binwang89** [comment]
@ltsheu Hi, I have tested it at both INT and QA and compared both site, at INT, i could use tab key to any error message and it will received focus. The QA will not h ave the latest code, and i can see the tab won't get to the error message.   it works fine now, I will move it to next pipeline. Thanks for the good work.
